### ISSUE TITLE
Support passing the code as a query param in the URL

### ISSUE THREAD

**Issue by PIG208** [radian-software/riju#182]
To quickly open the playground with some code, a useful feature would be passing the code text in the URL as a query parameter.

Let's say I have some python code:
```python
num = 42
print(f"Hello {num}")
```

I can base64 encode this code-block programmatically, so that I can generate a link to the playground with all the code.

**Comment by raxod502** [comment]
Yep, that makes sense. Just FYI, the build system for Riju has aged and it is not currently in a state where changes can be deployed, but as soon as I am able, I will be resolving that problem (which is actually a large horde of smaller problems) and making it possible to contribute again.
